x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/persistence/JobResultsPersister.java

```diff
@@ -411,4 +422,16 @@ private void logCall(String indexName) {
             }
         }
     }
+
+    public static class BulkIndexException extends Exception {
```

Comment:
Do we need both this exception class and the one defined in `ResultsPersisterService`?